gui/slick/interfaces/default/home.mako

```diff
@@ -814,6 +818,15 @@ $(document).ready(function(){
             </script>
         </td>
 
+        <% show_size = 0 %>
+        % try:
+            <% show_size = sickbeard.helpers.get_size(curShow.location) %>
+        % except sickbeard.exceptions.ShowDirNotFoundException:
+            <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        % endtry
+
```

Comment:
Kids are always right :)

I understand your point. What about trying to PEP8ing all the codebase too before that release? There is almost no PR left, so it shouldn't break too many things.